test/integration/tracked_watermark_buffer.cc

```diff
@@ -160,6 +164,38 @@ void TrackedWatermarkBufferFactory::removeDanglingAccounts() {
   }
 }
 
+void TrackedWatermarkBufferFactory::inspectAccounts(
+    std::function<void(AccountToBoundBuffersMap&)> func, Server::Instance& server) {
+  absl::Notification done_notification;
+  ThreadLocal::TypedSlotPtr<> slot;
+  Envoy::Thread::ThreadId main_tid;
+
+  server.dispatcher().post([&] {
```

Comment:
This seems more complicated than I expected.  Wondering if there's a way to simplify by capturing dispatchers associated with threads where accounts are created.